ServiceBus/ServiceBus/CampaignService.svc.cs

```diff
@@ -52,5 +52,11 @@ public Result DeleteCampaign(Guid guid)
             //before continuing is necessary to add Campaign to ServiceBusDatabaseEntities
             return Result.Fatal("Not Implemented");
         }
+
+        public Result AddCampaign(SharedLibs.DataContracts.Campaign campaign)
+        { 
+            //in order to compile 
+            throw new NotImplementedException();
```

Comment:
Do not throw Exceptions, rather create a result of type Fatal... return DataContracts.Result.Fatal("Not Implemented");